src/riak_index.erl

```diff
@@ -512,8 +512,10 @@ decode_continuation(Bin) ->
 %% We use `infinity' as the default to
 %% match the behavior pre 1.4
 add_timeout_opt(undefined, Opts) ->
```

Comment:
The above comment is now very misleading, because default is now `0`
It still holds, but now 0 is made into infinity. 
Probably explain this better in the comment